setup.py

```diff
@@ -29,40 +29,39 @@ def get_file_list(file_path):
 
 
 install_requires = [
-    "argon2_cffi>=20.1.0",
-    "beautifulsoup4[lxml]>=4.3.2",
-    "cbor2>=5.0.1",
-    "configobj>=5.0.6",
-    "croniter>=0.3.8",
-    "cryptography>=2.4.2",
-    "defusedxml>=0.4.1",
-    "Flask>=0.10.1,<2.0",
-    "Flask-Babel>=0.9,<3.0",
-    "Flask-Migrate>=1.2.0,<3.0",
-    "Flask-Script>=2.0.5",
-    "Flask-SQLAlchemy>=2.0,<3.0",
-    "Flask-Versioned>=0.9.4",
-    "google-auth>=1.23.0",
-    "grpcio>=1.46",
-    "huey[redis]>=1.11.0",
-    "importlib_metadata>=2.1.1; python_version=='3.7'",
-    "ldap3>=2.8,<2.9",
-    "MarkupSafe<2.1",
-    "netaddr>=0.7.12",
-    "passlib[bcrypt]>=1.7.0",
-    "protobuf>=3.20",
-    "pydash>=4.7.4",
-    "PyJWT>=1.3.0",
-    "PyMySQL>=0.6.6",
-    "pyOpenSSL>=17.5",
-    "pyrad>=2.0",
-    "python-dateutil>=2.7.3",
-    "python-gnupg>=0.4.4",
-    "PyYAML>=5.1",
-    "requests>=2.7.0",
-    "segno>=1.5",
-    "smpplib>=2.0",
-    "SQLAlchemy>=1.4.0,<2.0"
+    "argon2_cffi",
+    "beautifulsoup4[lxml]",
+    "cbor2",
+    "configobj",
+    "croniter",
+    "cryptography",
+    "defusedxml",
+    "Flask",
+    "Flask-Babel",
+    "Flask-Migrate",
+    "Flask-SQLAlchemy",
+    "Flask-Versioned",
+    "google-auth",
+    "grpcio",
+    "huey[redis]",
+    "importlib_metadata; python_version=='3.7'",
```

Comment:
python version 3.7 is no longer supported

Reply:
Done in 0091b57